Gemfile

```diff
@@ -57,6 +57,7 @@ group(:omnibus_package) do
   gem "chefspec"
   gem "fauxhai"
   gem "foodcritic", ">= 6.1.1"
+  gem "highline", ">= 1.6.9"
```

Comment:
sorry, I should have also noted that this should be in the `.gemspec` and not `Gemfile`